org.clojure/clojure/1.4.0/clj/clojure.core/iterate/notes.md

```diff
@@ -0,0 +1,13 @@
+## Arities
+
+    [f x]
+
+## Docstring
+
+Returns a lazy, infinite sequence `(x, (f x), (f (f x)), ...)`.
+
+## Note
+
+`f` must be free of side-effects. Not because this is enforced,
```

Comment:
Like all operations involving lazy sequences, `f` should be free of side effects.